OsmAnd/res/values-fr/phrases.xml

```diff
@@ -633,13 +633,13 @@
 	<string name="poi_transportation">Transport</string>
 	<string name="poi_man_made">Infrastructure</string>
 	<string name="poi_education">Éducation</string>
-	<string name="poi_administrative">Territoire</string>
+	<string name="poi_administrative">Administration</string>
 	<string name="poi_healthcare">Santé</string>
-	<string name="poi_office">Service</string>
+	<string name="poi_office">Tertiaire</string>
```

Comment:
Don't you think "Bureau" is very strange? I think nobody can understand 
which POI would be in such a category "Bureau": shops where you can buy 
pens? :-)


Le 12/05/2015 09:08, Stéphane Brunner a écrit :
>
> In OsmAnd/res/values-fr/phrases.xml 
> <https://github.com/osmandapp/Osmand/pull/1150#discussion_r30109299>:
>
> >  	<string name="poi_healthcare">Santé</string>
> > -	<string name="poi_office">Service</string>
> > +	<string name="poi_office">Tertiaire</string>
>
> Why not |Bureau| ?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/osmandapp/Osmand/pull/1150/files#r30109299>.
>